app/code/Magento/Directory/etc/adminhtml/system.xml

```diff
@@ -47,7 +47,12 @@
             </group>
             <group id="currencyconverterapi" translate="label" sortOrder="45" showInDefault="1" showInWebsite="0" showInStore="0">
                 <label>Currency Converter API</label>
-                <field id="timeout" translate="label" type="text" sortOrder="0" showInDefault="1" showInWebsite="0" showInStore="0">
+                <field id="api_key" translate="label" type="obscure" sortOrder="5" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
Please add validation as this field seems to be mandatory.

Reply:
User may input the API key for either Fixer.io or Currency Converter API on the backend, so not mandatory seems more user friendly.